articles/client-platforms/angularjs/01-login.md

```diff
@@ -1,118 +1,240 @@
 ---
-title: Auth0 Angular 1.x Tutorial
-description: This tutorial will show you how to use the Auth0 with Angular 1.x applications.
+title: Login
+description: This tutorial will show you how to use the Auth0 Angular SDK to add authentication and authorization to your mobile app.
 ---
 
-<%= include('../../_includes/_package2', {
-  org: 'auth0-samples',
-  repo: 'auth0-angularjs-sample',
-  path: '00-Starter-Seed'
+<%= include('../../_includes/_package', {
```

Comment:
We've updated the packager to version 2. Can you take a look at how I've configured the new packager [here](https://github.com/auth0/docs/blob/master/articles/client-platforms/angularjs/01-login.md) and follow that pattern for all of them?